packages/common/src/tools/LogThrottler.ts

```diff
@@ -0,0 +1,84 @@
+// eslint-disable-next-line import/no-cycle
+import { Logger } from './Logger'
```

Comment:
Instead of disabling `import/no-cycle` add an `ILogger` interface have `Logger implements ILogger` and `LogThrottler` receive `ILogger`